pkg/cluster_agent/tests/test_agent_deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
```

Comment:
Where is this deployment used?

Reply:
not used, deleting redundant file. thanks